.env.test

```diff
@@ -1,3 +1,4 @@
 # env vars for CI only
+IDP_URL=http://idp.example.com
 IDP_SSO_URL=http://idp.example.com/saml/auth
 IDP_SLO_URL=http://idp.example.com/saml/logout
```

Comment:
I don't like mixing implicit and explicit ENV vars like this. I think either use a root host env var or just call the new var IDP_SP_ENDPOINT